slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/SlashingProtectionFactory.java

```diff
@@ -22,16 +23,38 @@
 
 public class SlashingProtectionFactory {
 
+  public static final int EXPECTED_DATABASE_VERSION = 6;
```

Comment:
so, this would mean anytime we update our database scripts, we have to change the version here?

Reply:
yes, and the database script would need to perform an UPDATE on the version table